src/main/java/gregtech/integration/theoneprobe/provider/CoverInfoProvider.java

```diff
@@ -57,22 +58,25 @@ protected void addProbeInfo(@NotNull CoverHolder capability, @NotNull IProbeInfo
      * @param conveyor  the conveyor to get data from
      */
     private static void conveyorInfo(@NotNull IProbeInfo probeInfo, @NotNull CoverConveyor conveyor) {
-        String rateUnit = " {*cover.conveyor.transfer_rate*}";
+        String rateUnit = lang("cover.conveyor.transfer_rate");
```

Comment:
This class is a giant mess of my own creation. If you feel like it ought to be rewritten to accommodate the filter rework, go ahead. This can be postpones to a follow up PR if you think it's worthwhile to do at all as well.

Reply:
i do see some improvements that could be made here, but i'd want to do it in a separate pr since this is fine for now